cmd/state-svc/manager.go

```diff
@@ -31,6 +31,8 @@ type serviceManager struct {
 }
 
 func NewServiceManager(cfg *config.Instance) *serviceManager {
+	config.NewRule(constants.SvcConfigPid, config.Int, config.EmptyEvent, config.EmptyEvent)
+	config.NewRule(constants.SvcConfigPort, config.Int, config.EmptyEvent, config.EmptyEvent)
```

Comment:
This looks like the right package, but the wrong location. I think this should be inside an `init()` function as it's effectively a global setting. It doesn't seem worth passing this around in the config instance itself, though I'm open to being wrong about that. @daved thoughts?